src/pretix/base/migrations/0072_auto_20170803_1006.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2017-08-03 10:06
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def set_priority(app, schema_editor):
+    wles = app.get_model('pretixbase', 'WaitingListEntry')
+    for wle in wles.objects.all():
+        if wle.voucher:
+            wle.priority = None
+        else:
+            wle.priority = 10
+        wle.save()
```

Comment:
Number of Database Queries: 2 * #WaitlingListEntry + #WaitingListEntryWithVoucher

`wles.objects.filter(voucher__isnull=True).update(priority=10)`

More details:
- `wle.voucher` leads to a db query (check `wle.voucher_id` instead or use `select_related`)
- `wle.priority = None` is the default, no need to call save (=> hint that `filter` should be used)
- `wle.save()` sends all data to the database; consider using `save`'s `update_fields` instead